bin/zopen-init

```diff
@@ -131,283 +138,328 @@ while [ $# -gt 0 ]; do
+  echo "15" > "${rootfs}/etc/zopen/rm_fileprocs"
 
-export ZOPEN_ROOTFS="${rootfs}"
-printInfo "- Root FS available at ${rootfs}"
-# Generate any system values that can be user changed
-echo "15" > "${rootfs}/etc/zopen/rm_fileprocs"
+  if ${releaselineDev}; then
+    releaseLine="DEV"
+    printInfo "- Configured zopen to use development (DEV) releaseline packages where available" 
+  else
+    releaseLine="STABLE"
+    printInfo "- Configured zopen to use stable releaseline packages"
+  fi
+  echo "${releaseLine}" > "${rootfs}/etc/zopen/releaseline"
+}
 
-if ${releaselineDev}; then
-  releaseLine="DEV"
-  printInfo "- Configured zopen to use development (DEV) releaseline packages where available" 
-else
-  releaseLine="STABLE"
-  printInfo "- Configured zopen to use stable releaseline packages"
-fi
-echo "${releaseLine}" > "${rootfs}/etc/zopen/releaseline"
-
-printInfo "- Check for shipped curl pax"
-curlpax="packages/curl-8.4.0.20231011_155722.zos.pax.Z"
-
-curllcn=$(findrev "${MYDIR}" "${curlpax}")
-echo "${curllcn}/${curlpax}"
-if [ -e "${curllcn}/${curlpax}" ]; then
-  printInfo "- Found ${curlpax}. Adding to zopen cache to install as bootstrap"
-  cp -f "${curllcn}/${curlpax}" "${rootfs}/var/cache/zopen"
-else
-  printError "Could not locate curl pax in the zopen cache"
-fi
-printVerbose "- Expanding curl to bootstrap dir..."
-cachedir="${rootfs}/var/cache/zopen"
-basecurlpax=$(basename ${curlpax})
-[ -e "${cachedir}/${basecurlpax}" ] || printError "Could not locate bootstrap curl pax."
-paxrc=$(pax -rf "${cachedir}/${basecurlpax}" -s##${rootfs}/boot/#)
-
-printInfo "- Check for shipped jq pax"
-jqpax="packages/jq-1.6.20230720_160648.zos.pax.Z"
-
-jqlcn=$(findrev "${MYDIR}" "${jqpax}")
-if [ -e "${jqlcn}/${jqpax}" ]; then
-  printInfo "- Found ${jqpax}. Adding to zopen cache to install as bootstrap"
-  cp -f "${jqlcn}/${jqpax}" "${rootfs}/var/cache/zopen"
-else
-  printError "${jqpax} not found"
-fi
-printVerbose "- Expanding jq to bootstrap dir..."
-cachedir="${rootfs}/var/cache/zopen"
-basejqpax=$(basename ${jqpax})
-[ -e "${cachedir}/${basejqpax}" ] || printError "Could not locate bootstrap jq pax."
-paxrc=$(pax -rf "${cachedir}/${basejqpax}" -s##${rootfs}/boot/#)
-
-printVerbose "Configuring CA certificate file '${certFileName}'"
-cacertFound=false
-printVerbose "Check for existing ZOPEN_CA environment variable: ${ZOPEN_CA}"
-if [ -f "${ZOPEN_ROOTFS}/${ZOPEN_CA_DIR}/${certFileName}" ]; then
-  cacertFound=true
-elif [ -n "${ZOPEN_CA}" ]; then
-  printVerbose "Existing envvar for ZOPEN_CA found"
-  if [ -r "${ZOPEN_CA}" ]; then
-    printInfo "- Found existing setting for ZOPEN_CA '${ZOPEN_CA}'; reusing certificate"
-    if [ "${ZOPEN_CA}" != "${ZOPEN_ROOTFS}/${ZOPEN_CA_DIR}/${certFileName}" ]; then # avoid cp error
-      cp -f "${ZOPEN_CA}" "${ZOPEN_ROOTFS}/${ZOPEN_CA_DIR}"
-    fi
-    cacertFound=true
+installPrereqs()
+{
+  printInfo "- Check for shipped curl pax"
+  curlpax="packages/curl-8.4.0.20231011_155722.zos.pax.Z"
```

Comment:
should this be some sort of env var? presumably every time we get a CVE in curl we will need to update this?